srcpkgs/go-ethereum/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'go-ethereum'
+pkgname=go-ethereum
+version=1.8.18
+revision=1
+conflicts="geth"
+go_import_path="github.com/ethereum/go-ethereum"
+go_package="${go_import_path}/cmd/abigen"
+go_package+=" ${go_import_path}/cmd/bootnode"
+go_package+=" ${go_import_path}/cmd/evm"
+go_package+=" ${go_import_path}/cmd/geth"
+go_package+=" ${go_import_path}/cmd/puppeth"
+go_package+=" ${go_import_path}/cmd/rlpdump"
+go_package+=" ${go_import_path}/cmd/wnode"
+go_package+=" ${go_import_path}/cmd/swarm"
+build_style=go
+short_desc="Full suite of Go Ethereum utilities"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="GPL-3.0-only"
```

Comment:
Code inside `cmd` is GPL-3.0
Code outside `cmd` is LGPL-3.0

I've submitted fix to the Geth package https://github.com/void-linux/void-packages/pull/4902